engine/datahistory_manager.go

```diff
@@ -549,27 +559,27 @@ func (m *DataHistoryManager) validateJob(job *DataHistoryJob) error {
 		return errNilJob
 	}
 	if !job.Asset.IsValid() {
-		return asset.ErrNotSupported
+		return fmt.Errorf("job %s %w %s", job.Nickname, asset.ErrNotSupported, job.Asset)
 	}
 	if job.Pair.IsEmpty() {
-		return errCurrencyPairUnset
+		return fmt.Errorf("job %s %w", job.Nickname, errCurrencyPairUnset)
 	}
 	if !job.Status.Valid() {
-		return errInvalidDataHistoryStatus
+		return fmt.Errorf("job %s %w: %s", job.Nickname, errInvalidDataHistoryStatus, job.Status.String())
 	}
 	if !job.DataType.Valid() {
-		return errInvalidDataHistoryDataType
+		return fmt.Errorf("job %s %w: %s", job.Nickname, errInvalidDataHistoryDataType, job.DataType.String())
```

Comment:
pedantic: datatype and status both conform to the stringer interface so calling the method String() is not needed.